app/connector/rest-swagger/src/main/java/io/syndesis/connector/rest/swagger/auth/oauth/OAuthState.java

```diff
@@ -26,16 +26,30 @@
     private volatile State state;
 
     private static final class State {
-        private final String accessToken;
+        private final String _accessToken;
 
-        private final long accessTokenExpiresAt;
+        private final long _accessTokenExpiresAt;
 
-        private final String refreshToken;
+        private final String _refreshToken;
+
+        public synchronized String getAccessToken() {
+            return _accessToken;
+        }
+
+        public synchronized long getAccessTokenExpiresAt() {
+            return _accessTokenExpiresAt;
+        }
+
+        public synchronized String getRefreshToken() {
+            return _refreshToken;
+        }
```

Comment:
These are accessors for `final` fields of an `final` immutable class, why the need for locking?

Reply:
Same as above.